units/systemd-userdbd.service.in

```diff
@@ -13,6 +13,7 @@ Documentation=man:systemd-userdbd.service(8)
 Requires=systemd-userdbd.socket
 After=systemd-userdbd.socket
 Before=sysinit.target
+Wants=systemd-userdb-load-credentials.service
```

Comment:
Why not to add `After=` for the service?

Reply:
systemd-userdb-load-credentials already has `After=local-fs.target`, which means userdbd would be started a lot later. When I tried it it froze the boot process.

Reply:
I see. OK. Thanks.